### DISCUSSION TITLE
dependabot stuck on "Checking now" for days and `bin/dry-run.rb ...` looks good

### DISCUSSION THREAD

**Discussion by junwei-lii** [community/community#35283] [Other Feature Feedback, Questions, & Ideas] [answered]
- dependabot stuck on Gemfile "Checking now" for days
- tried https://github.com/dependabot/dependabot-core#dry-run-script in development machine, everything looks good

Pls help reset the checking job so our projects can use `dependabot` 🙏 

Suggestion: can we have a button or something to reset the check by ourselves ? Seeing it pending for days is annoying(we thought it was a long queue... )   :(

related:
- https://github.com/dependabot/dependabot-core/issues/5810
- https://github.com/orgs/community/discussions/35186

![image](https://user-images.githubusercontent.com/103160672/194195905-618b7db9-d0fb-4c54-a4ff-b48778c1db39.png)
------
![image](https://user-images.githubusercontent.com/103160672/194195950-2614d012-574e-40dc-aecd-a71d2c52c648.png)

**Comment by jeffwidman** [comment, marked as answer]
👋 HI, I work on the Dependabot team. 

This should be closed in favor of a private support ticket per my explanation here: https://github.com/dependabot/dependabot-core/issues/5810#issuecomment-1270358223
